plugins/pipelines/test/lib/hooks_test.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../test_helper'
+
+describe 'zendesk hooks' do
+  let(:deploy) { deploys(:succeeded_test) }
+  let(:next_deploy) { deploys(:succeeded_production_test) }
+  let(:stage) { deploy.stage }
+  let(:next_stages) { [ stages(:test_production), stages(:test_production_pod) ] }
+  let(:output) { StringIO.new }
+  let(:job) do
+    Job.create(project: stage.project,
+               command: "echo hello world",
+               status: "running",
+               user: User.first,
+               deploy: deploy)
+  end
+
+
+  describe :before_execute_finish_msg do
+    it 'kicks off the next stages in the deploy' do
```

Comment:
maybe best to have the default setup be the good case and then each test change 1 thing and then expect that it does not work